src/common.cpp

```diff
@@ -451,13 +451,23 @@ void setattr(PyObject *obj, PyObject *key, PyObject *value) {
 }
 
 void delattr(PyObject *obj, const char *key) {
+#if (defined(Py_LIMITED_API) && PY_LIMITED_API < 0x030D0000)
```

Comment:
I don't think we need the parens around the preprocessor expression (and this pattern generally isn't used in the codebase)